cmd/soroban-cli/src/commands/tx/send.rs

```diff
@@ -24,8 +25,11 @@ pub enum Error {
 #[derive(Debug, clap::Parser, Clone)]
 #[group(skip)]
 /// Command to send a transaction envelope to the network
-/// e.g. `cat file.txt | soroban tx send`
+/// e.g. `soroban tx send file.txt`
```

Comment:
Can we show both examples? For many of these commands piping is still the more common user flow.